common/fs/src/cache/dir_path.rs

```diff
@@ -89,6 +89,7 @@ impl From<DirPathBuf> for PathBuf {
 ///
 /// When a valid directory is found, it adds the missing piece of the directory
 /// path to `postfix` and returns the `DirPathBuf` result.
+#[cfg(unix)]
```

Comment:
@stsantilena 
i missed it - we want this feature (non-existing directories) tested on windows too.
possible?

Reply:
Not at this time. The fix for missing directory only applies to Linux at the moment. There is a ticket to extend to Windows.